src/server/services/hli/stadspas-router-external-consumer.test.ts

```diff
@@ -112,11 +112,14 @@ describe('hli/router-external-consumer', async () => {
         resMock
       );
 
-      expect(sendMock).toHaveBeenCalledWith({
-        status: 'ERROR',
-        message: 'Unauthorized',
-        content: null,
-      });
+      expect(renderMock).toHaveBeenCalledWith(
+        'amsapp-stadspas-administratienummer',
+        {
+          error: { code: '001', message: 'Niet ingelogd met Digid' },
+          appHref:
+            'amsterdam://stadspas?errorMessage=Niet ingelogd met Digid&errorCode=001',
```

Comment:
Kan hier wel met spaties gewerkt worden, moet dit niet misschien `%20` zijn in plaats van de spaties?